rundeckapp/grails-app/controllers/rundeck/controllers/UserController.groovy

```diff
@@ -445,7 +445,7 @@ Since: v21''',
             if(!updateResponse.isSaved){
                 def errorMsg= updateResponse.errors.allErrors.collect { g.message(error:it) }.join(";")
                 return apiService.renderErrorFormat(response,[
-                        status: HttpServletResponse.SC_INTERNAL_SERVER_ERROR,
+                        status: HttpServletResponse.SC_BAD_REQUEST,
```

Comment:
Should this be a 4XX error? it seems like this could legitimately be a 5XX depending on what happened.

Reply:
If `updateResponse.isSaved` is being checked and returns false it's because of a validation error with the data. If there's a severe problem and an exception is thrown in the save process this code wouldn't be reached.

Reply:
Ahh, gotcha.